packages/deployment-service/src/functions/pipeline/pipeline-setup.ts

```diff
@@ -134,7 +132,7 @@ export const pipelineSetup: SQSHandler = async (event: SQSEvent, context: Contex
                 },
               },
             ],
-            Comment: `Adding additional A record for pipeline [${pipeline.id}]`,
+            Comment: `Adding additional A record for pipeline [${pipeline.id}]  [${process.env.NODE_ENV === 'production' ? 'prod' : 'dev'}]`,
```

Comment:
And here

Reply:
same again: This is added to the lambda using env prop on lambda creation in cdk setup